subproject/simple/simple_dragonbox.h

```diff
@@ -0,0 +1,1996 @@
+#pragma once
+
+// Copyright 2024 Junekey Jeon, Toby Bell
+// TODO: License
+
+#include <assert.h>
+#include <limits.h>
+#include <stdint.h>
+#include <string.h>
+
+namespace simple_dragonbox {
```

Comment:
I think you should preserve the `detail` namespace and in the `simple_dragonbox` namespace only the public API's that are intended to be used by the users need to be exposed.